mcs/class/System/System.Diagnostics/Process.cs

```diff
@@ -302,7 +302,7 @@ ProcessModule[] GetModules_internal (SafeProcessHandle handle)
 		[DesignerSerializationVisibility (DesignerSerializationVisibility.Hidden)]
 		[MonitoringDescription ("The session ID for this process.")]
 		public int SessionId {
-			get { throw new NotImplementedException (); }
+			get { return 4; }
```

Comment:
Could you please change this to 0? Session 0 is the console session, so this should map most closely to Mono's use cases :)